website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -65,6 +65,43 @@ The most common types are:
  - `Gauge` Value can be increased and decreased as needed.
 5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
 
+#### Recommended Metrics
+
+{{% alert title="Warning" color="warning" %}}
+The following operator health metric is new and may be subjected to change 
+{{% /alert %}}
+* `<operator-name-prefix>_operator_health_status` - The metric reports the operator health status. It should be reported by the operator itself based on the internal operator validations.  
+  This metric allows the cluster users, to view all operators health statuses with a single metric and improves the overall cluster observability.
+
+  In order to maintain consistency, it is recommended to use the following health statuses as the metric values:
+  * `0` - Indicates that the operator is healthy and working as expected.
+  * `1` - Indicates that the operator has some issues that needs to be addressed and can potentially lead to loss of functionality.
+  * `2` - Indicates that the operator is unhealthy and there is a loss of functionality that should be addressed.
+
+  Metric Type: `Gauge`
+
+  Metric labels:
+  * `name` - The operator name [Recommended].
+  
+  Metric Help:
+  * Indicates whether the operator health status is healthy (0), warning (1) or critical (2).
+
+  **Example:**  
+```
+	desc := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name:      "memcached_operator_health_status",
+			Help:      "Indicates whether the operator health status is healthy (0), warning (1) or critical (2).",
+		},prometheus.Labels{"name": "Memcached"})
```

Comment:
```suggestion
			Name:      "operator_health_status",
			Help:      "Indicates whether the operator health status is healthy (0), warning (1) or critical (2).",
		},prometheus.Labels{"name": "memcached_operator_example_com"})
```

Reply:
The metric suggests to have the operator name as the prefix and the `operator_health_status` as the suffix. There example is the correct suggestion.

Reply:
The `name` label in intended for "Pretty" name. Not domain name.
This way the report can display the name and it will be easier for the user to identify the operator

Reply:
Ok, can we change the label name to "prettyName" then to make that obvious? Is that label required or optional?

Reply:
The name label is recommended, but is optional. I don;t see a reason to name it as "prettyName".
The idea is that in the metric itself the name prefix is short and this label can provide a more detailed name.